projects/flow.pytorch.HyperparameterOptimize/source/vae.py

```diff
@@ -0,0 +1,258 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
```

Comment:
Usually we don't import modules as single capital letters. How about importing it as `functional` or something more descriptive?

Reply:
I didn't write it and assumed it was a standard like `import numpy as np`, but I can change it.

Reply:
`nn` is already imported above it, so you could remove the import and use `nn.functional` instead of `F`